magma/lib/magma/attributes/collection.rb

```diff
@@ -7,7 +7,7 @@ def query_to_payload(link)
     end
 
     def query_to_tsv(value)
-      query_to_payload(value).join(", ")
+      value.join(", ")
```

Comment:
This fixes a bug in Magma and should already be in `master`. Pulled in because of `bin/pull-subtrees` I think.